### DISCUSSION TITLE
创建订单后返回状态码“100034(未知错误码)”和msg“票价发生变化”

### DISCUSSION THREAD

**Discussion by HYCF765** [mikumifa/biliTickerBuy#415] [提问专区] [answered]
今天测试抢票功能的时候遇到了没见过的问题，创建订单之后返回了状态码100034和 message 票价发生变化，但是从会员购页面上是能正常下单的。
换了一个会员购项目重新测试，订单顺利创建，但回到这个项目一直报这个错。网络没有挂梯子，抢票间隔设的2000ms。
是这个会员购项目有什么问题么？
![QQ截图20240826210711](https://github.com/user-attachments/assets/1ec0fdbb-280e-4b73-ab17-27121b9a9856)

**Comment by mikumifa** [comment, marked as answer]
下一下最新版本，已经解决这个问题了
